Src/BootCamp.Chapter/Lesson10.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    class Lesson10
```

Comment:
This whole class is not needed.
Avoid organizing code semantically (by layers, pieces, types and kinds.
Prefer to organize it logically (by the related functionality)

Reply:
I removed the whole Lesson 10 file and class